### PR TITLE
Update operator-crw-cr.yaml

### PR DESCRIPTION
**Pull request by gestrem**
add externalPluginRegistry: true parameter

### CODE REVIEW

File: cluster-init/ocp-4.4/operator-crw-cr.yaml [hunk 1]
```diff
@@ -8,6 +8,7 @@ spec:
     cheImageTag: ''
     cheFlavor: codeready
     devfileRegistryImage: ''
+    externalPluginRegistry: true # add it for CRW 2.3 on OCP 4.5 at least
```
Comment (lbroudoux) [inline cluster-init/ocp-4.4/operator-crw-cr.yaml:11]: Does this new attribute is related to an upgrade of the Custom Resource version ? Is it still compatible with CRW on OCP 4.4 ?

**Review by lbroudoux — COMMENTED** [review]
Just one comment on the new attribute

File: cluster-init/ocp-4.4/operator-crw-cr.yaml (hunk 1 above)
Comment (gestrem) [reply cluster-init/ocp-4.4/operator-crw-cr.yaml:11]: I will update the comment. It is not related OCP version.

**Comment by lbroudoux** [comment]
LGTM. Thanks!

**Pull request merged**